pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -145,6 +159,70 @@ func (v *vpnShoot) computeResourcesData() (map[string][]byte, error) {
 			},
 		}
 
+		serviceAccount = &corev1.ServiceAccount{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "vpn-shoot",
+				Namespace: metav1.NamespaceSystem,
+				Labels:    map[string]string{LabelKey: LabelValue},
+			},
+		}
+
+		deployment = &appsv1.Deployment{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      deploymentName,
+				Namespace: metav1.NamespaceSystem,
+				Labels: map[string]string{
+					v1beta1constants.GardenRole: v1beta1constants.GardenRoleSystemComponent,
+					LabelKey:                    LabelValue,
+					"origin":                    "gardener",
+				},
+				//TODO: Add annotations
+				//Annotations: ,
+			},
+			Spec: appsv1.DeploymentSpec{
+				RevisionHistoryLimit: pointer.Int32(1),
+				Replicas:             pointer.Int32(1),
+				Strategy: appsv1.DeploymentStrategy{
+					Type: appsv1.RollingUpdateDeploymentStrategyType,
+					RollingUpdate: &appsv1.RollingUpdateDeployment{
+						MaxSurge:       &intStrOne,
+						MaxUnavailable: &intStrZero,
+					},
+				},
+				Selector: &metav1.LabelSelector{
+					MatchLabels: map[string]string{LabelKey: LabelValue},
+				},
+				Template: corev1.PodTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						Annotations: utils.MergeStringMaps(v.values.PodAnnotations, map[string]string{
+							"scheduler.alpha.kubernetes.io/critical-pod": "",
```

Comment:
You can remove this - it is a left-over and no longer needed for the Kubernetes versions Gardener supports these days.